src/content/api/compilation-hooks.md

```diff
@@ -464,6 +465,50 @@ The assets have been optimized.
 - Callback Parameters: `assets`
 
 
+### `processAssets`
+
+`AsyncSeriesHook`
+
+Asset processing.
+
+- Callback Parameters: `assets`
+
+Here's an example:
+
+```js
+compilation.hooks.processAssets.tap(
+  {
+    name: 'MyPlugin',
+    stage: Compilation.PROCESS_ASSETS_STAGE_ADDITIONS,
+  },
+  (assets) => {
+    // code here
+  }
+);
+```
+
+There're many stages to use:
+
+- `PROCESS_ASSETS_STAGE_ADDITIONAL` - Add additional assets to the compilation.
+- `PROCESS_ASSETS_STAGE_PRE_PROCESS` - Basic preprocessing of assets.
+- `PROCESS_ASSETS_STAGE_DERIVED` - Derive new assets from existing assets.
+- `PROCESS_ASSETS_STAGE_ADDITIONS` - Add additional sections to existing assets, like a banner or initialization code.
+- `PROCESS_ASSETS_STAGE_OPTIMIZE` - Optimize existing assets in a general way.
+- `PROCESS_ASSETS_STAGE_OPTIMIZE_COUNT` - Optimize the count of existing assets, e.g. by merging them.
+- `PROCESS_ASSETS_STAGE_OPTIMIZE_COMPATIBILITY` - Optimize the compatibility of existing assets, e.g. add polyfills or vendor-prefixes.
```

Comment:
I think `vendor prefixes` is more common than `vendor-prefixes`, i'd suggest to go with the common version
```suggestion
- `PROCESS_ASSETS_STAGE_OPTIMIZE_COMPATIBILITY` - Optimize the compatibility of existing assets, e.g. add polyfills or vendor prefixes.
```